WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Dynamic/DynamicDashboardCard.swift

```diff
@@ -0,0 +1,143 @@
+import SwiftUI
+import DesignSystem
+
+struct DynamicDashboardCard: View {
+    private enum Constants {
+        static let featureImageHeight: CGFloat = 151
+        static let rowImageDiameter: CGFloat = 48
+    }
+
+    struct Input {
+        struct Row: Identifiable {
+            let id: UUID = UUID()
```

Comment:
@alpavanoglu I could be missing something but this could be the dynamic card `id` that comes from the Backend.

Reply:
Oh right! I had forgotten about it. Let's do that.

Reply:
@salimbraksa Actually, I jumped on it too quickly. The ID here is for the Row and not the whole card so that we can use `SwiftUI.ForEach` on the rows array. So it isn't connected to the Card ID from the backend.